packages/fx-core/templates/plugins/resource/apim/README.md

```diff
@@ -34,7 +30,7 @@ Simply deploy your project to the cloud when it’s ready by following these ste
 - Deploy to the cloud
 
 You can do this using the Teams Toolkit in Visual Studio Code or using the TeamsFx CLI:
-| Using Teams Toolkit| Using TeamsFx CLI|
+| From Visual Studio Code | From TeamsFx CLI|
```

Comment:
If you change `Using` to `From`, do we need to update the same wording in above section?

Reply:
Change it same to the tab app readme